tests/semanticdb/expect/Givens.expect.scala

```diff
@@ -4,11 +4,11 @@ package b
 object Givens/*<-a::b::Givens.*/:
 
   extension [A/*<-a::b::Givens.sayHello().[A]*/](any/*<-a::b::Givens.sayHello().(any)*/: A/*->a::b::Givens.sayHello().[A]*/)
-    def sayHello/*<-a::b::Givens.sayHello().*/ = s"Hello, I am $any/*->a::b::Givens.sayHello().(any)*/"
+    def sayHello/*<-a::b::Givens.sayHello().*/ = s"Hello, I am $any/*->a::b::Givens.sayHello().(any)*/"/*->scala::StringContext#s().*/
```

Comment:
Introducing `StringContext#s` is fine, because

- it actually presents in the source
- Scala2 also extracts it https://github.com/scalameta/scalameta/blob/267c5966bc343ee36683ca36bd19cf6e9b825381/tests/jvm/src/test/resources/example/ImplicitConversion.scala#L23

Though the position is a bit wired 🤔

Reply:
if i compile
```scala
def str = s"abc 123 ${scala.util.Random.nextBoolean}"
```
the span of `.s` in the desugared code is `10-53`, so the span comes from somewhere and is truncated, maybe `selectSpan` or `nameSpan`?

either way it does not need to be fixed in this PR